test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -73,6 +73,106 @@ def test_successful_3ds2_authorize
     assert_equal 'SUCCESS', response.message
   end
 
+  def test_successful_authorize_with_risk_data
+    authentication_time = Time.now
+    shopper_account_creation_date = Date.today
+    shopper_account_modification_date = Date.today - 1.day
+    shopper_account_password_change_date = Date.today - 2.days
+    shopper_account_shipping_address_first_use_date = Date.today - 3.day
+    shopper_account_payment_account_first_use_date = Date.today - 4.day
+    transaction_risk_data_pre_order_date = Date.today + 1.day
+
+    risk_data = {
```

Comment:
This is a lot of setup for this test (and the unit one, too). Are there standard/prettier ways that we'd usually do this sort of thing?

Reply:
Yeah, I looked around at other XML gateways and didn't see anything like this, so it _felt_ like maybe I was going down the wrong path doing it like this... 🤔

Reply:
We do usually set up test values for a GSF when we add one. We just don't typically add _this many_ GSFs all at once!

I do think that moving this setup to a separate private method would help things look tidier, though.